File: ThemisBlazorApp2.0/ThemisBlazorApp2.0/Data/DataTransferObjects/BackLogItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ThemisBlazorApp2._0.Data.DataTransferObjects
{
    public class BackLogItem
    {
        public int Id;
        public String Priority { get; set; }
        public String Description { get; set; }
        public int EstimatedTime { get; set; }
        public String Status { get; set; }
        public User WorksOnIt { get; set; }
        public int ActualTime { get; set; }

        public BackLogItem()
        {
        }

        public BackLogItem(string priority, int estimatedTime, string description)
        {
            this.Priority = priority;
            this.EstimatedTime = estimatedTime;
            this.Description = description;
        }
    }
}

File: Logic_API/src/main/java/dataTransferObjects/BackLogItem.java
package dataTransferObjects;

public class BackLogItem
{
    private int Id;
    private String Priority;
    private String Description;
    private int EstimatedTime;
    private String Status;
    private User WorksOnIt;
    private int ActualTime;

    public BackLogItem()    {
    }

    public BackLogItem(String priority, String status, int estimatedTime, int actualTime)    {
        this.Priority = priority;
        this.Status = status;
        this.EstimatedTime = estimatedTime;
        this.ActualTime = actualTime;
    }

    public BackLogItem(String description, String status, int estimatedTime)    {
        Description = description;
        Status = status;
        EstimatedTime = estimatedTime;
    }

    public BackLogItem(int id, String priority, String description, int estimatedTime, String status, User worksOnIt, int actualTime)    {
        Id = id;
        Priority = priority;
        Description = description;
        EstimatedTime = estimatedTime;
        Status = status;
        WorksOnIt = worksOnIt;
        ActualTime = actualTime;
    }

    public String getPriority()
    {
        return Priority;
    }

    public void setPriority(String priority)
    {
        Priority = priority;
    }

    public String getStatus()
    {
        return Status;
    }

    public void setStatus(String status)
    {
        Status = status;
    }

    public int getEstimatedTime()
    {
        return EstimatedTime;
    }

    public void setEstimatedTime(int estimatedTime)
    {
        EstimatedTime = estimatedTime;
    }

    public int getActualTime()
    {
        return ActualTime;
    }

    public void setActualTime(int actualTime)
    {
        ActualTime = actualTime;
    }

    public int getId()
    {
        return Id;
    }

    public void setId(int id)
    {
        Id = id;
    }

    public String getDescription()
    {
        return Description;
    }

    public void setDescription(String description)
    {
        Description = description;
    }

    public User getWorksOnIt()
    {
        return WorksOnIt;
    }

    public void setWorksOnIt(User worksOnIt)
    {
        WorksOnIt = worksOnIt;
    }
}